### PR TITLE
Add seed2 test

### PR DESCRIPTION
**Pull request by WallE256**

### CODE REVIEW

File: seed2/tests/cpp_ac/seed2.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        if(res == "OK")
+            return 0;
+        else if(res == "FLOATS")
+            high = mid-1;
+        else
+            low = mid+1;
+    }
+    cout << low << endl;
+    return 0;
+}
```
Comment (Xyene) [inline seed2/tests/cpp_ac/seed2.cpp]: newline

File: seed2/tests/cpp_ac/seed2.cpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#include <bits/stdc++.h>
+
+using namespace std;
+
+int main(int argc, const char * argv[]) {
+    cin.tie(0); 
+    cin.sync_with_stdio(0);
+    long long low = 1, high = 2000000000;
+    while(low <= high){
```
Comment (Xyene) [inline seed2/tests/cpp_ac/seed2.cpp]: spacing etc and elsewhere

File: seed2/tests/java_ac/seed2.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import java.util.*;
+public class Seed2 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
```
Comment (Xyene) [inline seed2/tests/java_ac/seed2.java]: indents

File: seed2/tests/tur_ac/seed2.tur [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    mid := minium + (maxium - minium) div 2
+    put mid
+    get status
+    if status = "SINKS" then
+        minium := mid + 1
+    elsif status = "FLOATS" then
+        maxium := mid - 1
+    end if
+    exit when status = "OK"
+end loop
```
Comment (Xyene) [inline seed2/tests/tur_ac/seed2.tur]: eof nl

File: seed2/tests/java_ac/seed2.java [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import java.util.*;
+
+public class Seed2 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String result;
+        long lo = 1;
+        long hi = 2000000000;
+        long num;
+        while(lo < hi) {
```
Comment (quantum5) [inline seed2/tests/java_ac/seed2.java]: Spacing of this file is still terrible. Try an autoformatter?

**Pull request merged**